cpp/harvest/remid_augment.cc

```diff
@@ -20,63 +20,69 @@
 #include <fstream>
 #include <iostream>
 #include <string>
+#include "FileUtil.h"
 #include "MARC.h"
 
 namespace {
 
 [[noreturn]] void Usage() {
-    std::cerr << "Usage: " << ::progname << " marc_input marc_output\n";
+    std::cerr << "Usage: " << ::progname << "  marc_input marc_output issn_output\n";
     std::cerr << "       marc_input is the marc input file\n";
-    std::cerr << "       marc_output is the marc output file\n";
+    std::cerr << "       marc_output is the marc output file without serial\n";
+    std::cerr << "       issn_output is the text output file for issn that ha\n";
```

Comment:
Regarding `issn_output`: This is technically wrong, because the output file will not contain the ISSNs => it will contain the ZDB IDs. I think it would be better to rename the parameter to `journal_output` or `serial_output`. The usage should then state that the journal_output contains the ZDB IDs as a text file and that all journals will be removed from the regular marc_output file.